### ISSUE TITLE
3. Update ratio of affordable and available housing units to households with low-, very low-, and extremely low-income levels

### ISSUE THREAD

**Issue by awunderground** [UI-Research/mobility-from-poverty#202]
Pillar: Opportunity-Rich & Inclusive Neighborhoods
Predictor: Housing Affordability
Metric: Ratio of affordable and available housing units to households with low, very low, and extremely low income levels

- [ ] Rename to `share_affordable_30_ami`, `share_affordable_50_ami`, `share_affordable_80_ami`, and `share_affordable_quality`.
- [ ] Add subgroup for rent vs. own
- [ ] Add 2022 data

**Comment by cdsolari** [comment]
This is to clarify how we present the new metric within affordable housing for the "affordable & available." Like our existing data tables, the new dashboard will distinguish an overall value and subgroups. It makes most sense to stay consistent with the literature and how these metrics are presented and used as it relates to the Worst Case Housing Needs Report (e.g., most recent version: https://www.huduser.gov/portal/portal/sites/default/files/pdf/Worst-Case-Housing-Needs-2023.pdf) is to have the affordable for the very low-income households value show and to have "affordable & available" show for that same income group. When we go to the subgroups, we will expand these to the two other income groups (Extremely low income (30% AMI) and low income (80% AMI)). This is important because it impacts the output file structure. 

I will continue to look into how individual communities tend to use this and if I see they use it differently than the WCHN reports, we can re-assess. But let's plan for this approach, if that is feasible at this juncture.

**Comment by cdsolari** [comment]
In a meeting, we discussed some edits to the overall and subgroup files. The overall file would have both share_affordable for just one of the AMI categories (and the quality flag), and the affordable_available for the same AMI category. We discussed having the 50_ami. Upon reflection, it might make more sense to have the 30_ami group as the focal group because that might resonate with people the most. You mentioned for the affordable_available that you used the prefix rate. Upon reflection, these two metrics should have the same prefix. I think it should either be "share_" or "ratio_" for both. For the subgroup, you will have the other two ami groups, along with the renter/owner difference. We spoke about having the denominator for the renter to just be renters and the denominator for owner to just be among owners. Chris's CHAS code might have a flag for the denominator for renters. We are doing this because people are really used to seeing this figure for just renters and not owners. If not that, we should have a denominator variable in the Seattle folder within the metrics 2024 folder. Please let me know if you're having trouble. I can try to help. Thanks!

**Comment by cdsolari** [comment]
on 2/8, we decided that every value reported should have a data quality index value. Most metrics already do, but like income at 20th, 50th, and 80th percentiles, the affordable housing metric has values at the 30_ami, 50_ami, and 80_ami. In the past, we reported only one data quality index value for the set of 3, but we realize that it is most appropriate to consider quality for each. This means adding several variables where each of the ami levels will have a 30_ami_quaity, 50_ami_quality, and 80_ami_quality. Also because we have 2 measures for housing affordability: affordable housing units per 100 households AND affordable & available housing units per 100 households, we need quality flags for each of these categories for both measures. These quality indicators will extend to the subgroups file such that a value for quality will be needed among renters and among owners. 

We should look again at how data quality was determined to see if we need to alter our approach. I imagine that some places have a smaller number of units for extremely low-income households vs higher income households. And, I imagine we might have much smaller sample sizes as we consider owner-units vs rental-units.
